### PR TITLE
bg: add French translation

### PR DESCRIPTION
**Pull request by julio4**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/bg.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# bg
+
+> Reprend l'exécution de tâches qui ont été suspendues (en utilisant `Ctrl + Z` par exemple) en arrière-plan.
+
```
Comment (kbdharun) [inline pages.fr/common/bg.md:4]:
```suggestion
- More information: https://manned.org/bg.

```

Comment (julio4) [reply pages.fr/common/bg.md:4]: Should we keep links to a copy of man pages ?  I thought that if a user needed more information he would use `man bg`, so I don't see the point to keep a link to manned.

Comment (kbdharun) [reply pages.fr/common/bg.md:4]: Usually, we use manned as a fallback when there isn't a man page around. This link is used in `bg` English page so it is better to have it here for better alignment between pages.

Comment (julio4) [reply pages.fr/common/bg.md:4]: Ok, I understand!

**Review by kbdharun — APPROVED** [review]

File: pages.fr/common/bg.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# bg
+
+> Reprend l'exécution de tâches qui ont été suspendues (en utilisant `Ctrl + Z` par exemple) en arrière-plan.
+> Plus d'informations: <https://manned.org/bg>.
```
Comment (navarroaxel) [inline pages.fr/common/bg.md:4]: We use the French space
```suggestion
> Plus d'informations : <https://manned.org/bg>.
```

File: pages.fr/common/bg.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+- Reprend l'exécution de la dernière tâche suspendue en arrière-plan:
```
Comment (navarroaxel) [inline pages.fr/common/bg.md:6]:
```suggestion
- Reprend l'exécution de la dernière tâche suspendue en arrière-plan :
```

File: pages.fr/common/bg.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+`bg`
+
+- Reprend l'exécution d'une tâche précise (utiliser `jobs -l` pour obtenir son ID) en arrière-plan:
```
Comment (navarroaxel) [inline pages.fr/common/bg.md:10]:
```suggestion
- Reprend l'exécution d'une tâche précise (utiliser `jobs -l` pour obtenir son ID) en arrière-plan :
```

**Comment by julio4** [comment]
@navarroaxel Nice catch, thank you!

**Review by kbdharun — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**